neuralhydrology/utils/config.py

```diff
@@ -397,6 +397,14 @@ def learning_rate(self) -> Dict[int, float]:
         else:
             raise ValueError("No learning rate specified in the config (.yml).")
 
+    @property
+    def learning_rate_drop_factor(self) -> float:
+        return self._cfg.get("learning_rate_drop_factor", None)
+
+    @property
+    def learning_rate_epochs_drop(self) -> int:
+        return self._cfg.get("learning_rate_epochs_drop", None)
+
```

Comment:
As stated in the other comment, I am in favor of not adding the lr scheduling in this PR.